src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import './Navabar.css';
+import search from '../../images/searchicon.svg';
+import home from '../../images/home.svg';
+
+function Navbar({ pageTo }) {
+  switch (pageTo) {
+    case 'No':
+      return <div>No</div>;
+    default:
+      return (
+        <div className="navbar">
+          <Link to="/"><img className="navbar__img" src={home} alt="Home" /></Link>
+          <h1 className="navbar__title">COVID-19 stats</h1>
+          <img className="navbar__img" src={search} alt="search" />
```

Comment:
- [ ] It would be better if you can filter categories using given parameters, e.g. category name. kindly add this feature as one of the requirements. 

![Screenshot from 2022-04-21 13-22-30](https://user-images.githubusercontent.com/64173189/164528080-abc9777f-954a-412b-91a9-f7e4c54d51f6.png)